pages/advocates/index.vue

```diff
@@ -121,6 +123,10 @@ import CompactFeature from '~/components/features/CompactFeature.vue'
   }
 })
 export default class extends Vue {
+  title: string = 'Qiskit Advocates'
+  belongsTo: string = 'advocates'
```

Comment:
Assuming this string matches the folder name that defines the URL, I would bet this information is in the router since the Nuxt router [uses the folder hierarchy in `pages`](https://nuxtjs.org/guide/routing/) to create its hierarchy.